tests/test_signals.py

```diff
@@ -146,5 +149,5 @@ def do_something_on_page_publish_unpublsh(*args, **kwargs):
 
         # Only the publish and unpublish signals should have had an affect
         self.assertEqual(len(signal_hits), 2)
-        self.assertEqual(signal_hits[0].state, constants.UNPUBLISHED)
-        self.assertEqual(signal_hits[1].state, constants.UNPUBLISHED)
+        self.assertEqual(signal_hits[0].get('state'), constants.PUBLISHED)
```

Comment:
Why `.get('state')` and not just `['state']`? In a test it's usually better to be as specific as possible. If things break in a test, it's best if they break as loudly as possible.

Reply:
.get('state') will return None which would fail the check.